### PR TITLE
tutorials~~~

### PR DESCRIPTION
**Pull request by SoluMilken**
The tutorials of how to use various evaluation tools provided by YNLU.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,8 @@
         "matplotlib==2.2.2",
         "seaborn==0.8.1",
         "sklearn==0.0",
+        "ipython==6.2.1",
+        "jupyter==1.0.0",
```
Comment (stegben) [inline setup.py]: these shouldn't be requirements, simply add them to `requirements.txt`

Comment (SoluMilken) [reply setup.py]: :ok_woman:

Comment (SoluMilken) [reply setup.py]: 94d67f7

**Comment by SoluMilken** [comment]
還有啥要加加？？ :thinking:

**Comment by stegben** [comment]
The ipython notebook filenames don't seems valid on some platform. Plz use snake case

**Comment by SoluMilken** [comment]
:snake: case ???

**Comment by SoluMilken** [comment]
OHOHOH https://en.wikipedia.org/wiki/Snake_case :snake:  :snake:  :snake:

**Comment by SoluMilken** [comment]
Need some suggestions about tutorials to be added. :place_of_worship:

**Pull request merged**